fix.sh

```diff
@@ -111,6 +111,12 @@ xmlstarlet ed -L \
 	-u '//_:type[@c:type="hb_feature_t*"]/@c:type' -v "gconstpointer" \
 	Pango-1.0.gir
 
+# Fix broken bitfield on pango
```

Comment:
It's not broken, just not supported by gir :) Also I suggested to replace the content of `PangoGlyphVisAttr` accordingly. That should be sufficient.

Reply:
Right, I rephrased the comment. Although, I'm not sure why you want to replace the whole content of `PangoGlyphVisAttr` ?

Reply:
It's only used in GlyphInfo btw

Reply:
Yes but like that you would keep the structure of the types at least

Reply:
Right, but I'm unsure how to handle it then. What I'm supposed to modify in the VisAttr?

Reply:
never mind, should be fixed now